cmd/generate_keygen.go

```diff
@@ -39,7 +39,7 @@ func keygenMain(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "failed to get the current working directory")
 	}
 	if privateKeyPath == "" {
-		privateKeyPath = filepath.Join(wd, "issuer.jwk")
+		privateKeyPath = filepath.Join(wd, "issuer-keys")
```

Comment:
Actually, the private/public generation logic lies in `config.GetIssuerPublicJWKS()`, which was already updated to incorporate the new keys in IssuerKeysDirectory instead of only one key file at IssuerKey